### PR TITLE
IGAPP-966: Fix jira releases

### PR DESCRIPTION
**Pull request by klinzo**
This pull request belongs to an issue on our [bugtracker](https://issues.integreat-app.de/).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword **IGAPP**.

### CODE REVIEW

File: tools/jira-release.ts [hunk 1]
```diff
@@ -0,0 +1,138 @@
+import { program } from 'commander'
```
Comment (klinzo) [inline tools/jira-release.ts:1]: Converted to typescript and converted manual oauth get/post/put with jira api calls.

**Comment by klinzo** [comment]
I already tested this, should work.

File: .circleci/config.yml [hunk 2]
```diff
@@ -490,8 +490,9 @@ jobs:
                         name: Bump git version
                         working_directory: tools
                     - run:
-                        command: tools/create-jira-release create-release ${NEW_VERSION_NAME} --private-key ${JIRA_PRIVATE_KEY} --access-token ${JIRA_OAUTH_TOKEN} --consumer-key ${JIRA_CONSUMER_KEY} --project-name ${CIRCLE_PROJECT_REPONAME}
+                        command: yarn jira-release create ${NEW_VERSION_NAME} --private-key ${JIRA_PRIVATE_KEY} --access-token ${JIRA_OAUTH_TOKEN} --consumer-key ${JIRA_CONSUMER_KEY} --project-name ${CIRCLE_PROJECT_REPONAME}
```
Comment (SteffiStuffel) [inline .circleci/config.yml:493]: create-jira-release would be a better naming.

**Review by SteffiStuffel — APPROVED** [review]
Looks good.

File: .circleci/config.yml (hunk 2 above)
Comment (klinzo) [reply .circleci/config.yml:493]: Imo not otherwise you have `create-jira-release create` which is duplicated.

Comment (SteffiStuffel) [reply .circleci/config.yml:493]: you are right. overlooked it.

**Review by f1sh1918 — APPROVED** [review]

**Pull request merged**